index.html

```diff
@@ -418,6 +419,145 @@ <h2>Transferable MediaStreamTrack</h2>
+        <li>
+          <p>Atomically increment <var>track</var>.`[[FramesCaptured]]` by
+          1.</p>
+        </li>
+      </ol>
+    </div>
+    <section>
+      <h4>Methods</h4>
+      <dl data-link-for="MediaStreamTrack" data-dfn-for="MediaStreamTrack"
+      class="methods">
+        <dt>
+          <dfn>getStats</dfn>
+        </dt>
+        <dd>
+          <p>
+            When this method is called, the user agenst MUST run the following
+            steps:
+          </p>
+          <ol>
+            <li>
+              <p>
+                Let <var>track</var> be the {{MediaStreamTrack}} that this
+                method was called on.
+              </p>
+            </li>
+            <li>
+              <p>Let <var>p</var> be a new promise.</p>
+            </li>
+            <li>
+              <p>
+                In parallel, begin the [= in-parallel steps to get track
+                stats =], given <var>track</var> and <var>p</var>.
+              </p>
+            </li>
+            <li>
+              <p>Return <var>p</var>.</p>
+            </li>
+          </ol>
+          <p>
+            The <dfn>in-parallel steps to get track stats</dfn> given
+            <var>track</var> and <var>p</var> are as follows:
+          </p>
+          <ol>
+            <li>
+              <p>
+                Let <var>timestamp</var> be the current time relative to the
+                UNIX epoch (Jan 1, 1970, UTC).
+              </p>
+            </li>
+            <li>
+              <p>
+                Let <var>framesCaptured</var> be the result of atomically
+                reading <var>track</var>.`[[FramesCaptured]]`.
+              </p>
+            </li>
+            <li>
+              <p>
+                Queue a task to resolve <var>p</var> with a newly constructed
+                {{MediaStreamTrackStats}} dictionary where
+                {{MediaStreamTrackStats/timestamp}} is set to
+                <var>timestamp</var> and
+                {{MediaStreamTrackStats/framesCaptured}} set to
+                <var>framesCaptured</var>.
+              </p>
+            </li>
+          </ol>
+        </dd>
+      </dl>
+    </section>
+    <section>
+      <h4>The MediaStreamTrackStats dictionary</h4>
+      <div>
+        <pre class="idl" data-cite="HR-TIME"
+>dictionary MediaStreamTrackStats {
+  DOMHighResTimeStamp timestamp;
+  unsigned long long framesCaptured;
+};
+</pre>
+        <section>
+          <h4>Dictionary {{MediaStreamTrackStats}} Members</h4>
+          <dl data-link-for="MediaStreamTrackStats"
+          data-dfn-for="MediaStreamTrackStats" class="dictionary-members">
+            <dt>
+              <dfn data-idl="">timestamp</dfn> of type <span class=
+              "idlMemberType">DOMHighResTimeStamp</span>
+            </dt>
+            <dd>
+              <p>
+                The {{timestamp}}, relative to the UNIX epoch (Jan 1, 1970, UTC),
+                for when these stats where collected.
```

Comment:
This leaves the question of "on what clock" somewhat up in the air (monotonic, requires timesync...)

In RTCRtpContributingSources, we defined timestamp as Performance.timeOrigin + Performance.now() - someone claimed that Performance.now() doesn't advance while computer is suspended, so this may be the wrong definition to use, but we should strive to at least make the same errors as everyone else.

Reply:
I can't find anything about Performance.now() pausing while suspended, but perhaps we can simply reference the [shared monotonic clock](https://w3c.github.io/hr-time/#dfn-shared-monotonic-clock) from the High Resolution Time spec, which would remove any dependency on current document or stuff like that

Reply:
Good find. That means that if Performance.now() isn't incremented while suspended, it's a bug.

Reply:
OK then "Performance.timeOrigin + Performance.now()" is probably still best